backend/monitoring/dashboard.py

```
@@ -707,6 +732,127 @@ async def trigger_test(test_type: str, admin=Depends(get_current_admin_dependenc
+                "auto_fixes_applied": 2
+            }
+        )
+    except Exception as e:
+        return StandardResponse(
+            status="error",
+            message=f"Failed to get test status: {str(e)}",
+            data={}
+        )
+
+@router.get("/test-sessions")
+async def get_test_sessions(admin=Depends(get_current_admin_dependency)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)